pkg/daemon/ceph/client/subvolumegroup.go

```diff
@@ -129,3 +131,77 @@ func validatePinningValues(pinning cephv1.CephFilesystemSubVolumeGroupSpecPinnin
 	}
 	return err
 }
+
+func GetOMAPKey(context *clusterd.Context, clusterInfo *ClusterInfo, omapObj, poolName, namespace string) (string, error) {
```

Comment:
These are the same methods as the kubectl plugin was using, so the plugin will plan on calling these?